quadriga/src/main/java/edu/asu/diging/quadriga/core/db/impl/AppDbConnection.java

```diff
@@ -0,0 +1,74 @@
+package edu.asu.diging.quadriga.core.db.impl;
+
+import java.util.Random;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.transaction.Transactional;
+
+import org.springframework.stereotype.Component;
+
+import edu.asu.diging.quadriga.core.db.IAppDbConnection;
+import edu.asu.diging.quadriga.core.exceptions.UnstorableObjectException;
+import edu.asu.diging.quadriga.core.model.IApp;
+import edu.asu.diging.quadriga.core.model.impl.App;
+
+@Component
+@Transactional
+public class AppDbConnection implements IAppDbConnection {
+
+    @PersistenceContext
+    private EntityManager em;
```

Comment:
you should use repositories instead of the entitymanager; this class is probably not necessary at all, just use the repositories from the manager classes.

Reply:
?